src/bindings/python/src/pyopenvino/frontend/onnx/extension.cpp

```diff
@@ -41,3 +42,37 @@ void regclass_frontend_onnx_ConversionExtension(py::module m) {
 
     ext.def_property_readonly_static("m_converter", &ConversionExtension::get_converter);
 }
+
+void regclass_frontend_onnx_OpExtension(py::module m) {
+    py::class_<OpExtension<void>, std::shared_ptr<OpExtension<void>>, ConversionExtension> ext(
+            m,
+            "OpExtensionONNX",
```

Comment:
Why do we have implemented only `OpExtensionONNX` only? What is about TF and PDPD? Or why don't we have common implementation for all frontends?

Reply:
not only onnx, we also have OpExtensionTensorflow and frontend::OpExtension (common)
PaddlePaddle has a specific, we need to map named inputs and named outputs to ov inputs/outputs, so we can't use it without additional information from user, e.g. edges

Reply:
see pyopenvino/frontend/extension.cpp/hpp
pyopenvino/frontend/tensorflow/extension.cpp/hpp